_config.yml

```
@@ -40,6 +40,8 @@ code_variations:
     folder: "Processing"
   - name: "JavaScript"
     folder: "JavaScript"
+  - name: "NODE.JS"
```

Is there a reason why this is all capital?
```suggestion
  - name: "Node.js"
```

---

There'll no difference. The style will capitalize it anyway.